l1-contracts/contracts/bridge/L1SharedBridge.sol

```diff
@@ -85,18 +86,18 @@ contract L1SharedBridge is IL1SharedBridge, ReentrancyGuard, Ownable2StepUpgrade
 
     /// @dev Maps token balances for each chain to prevent unauthorized spending across hyperchains.
     /// This serves as a security measure until hyperbridging is implemented.
-    mapping(uint256 chainId => mapping(address l1Token => uint256 balance)) internal chainBalance;
+    mapping(uint256 chainId => mapping(address l1Token => uint256 balance)) public chainBalance;
```

Comment:
One thing about this is that as I understand this is a temporary measure until we enable hyperbridging. 
So we will still have to support it afterwards if we make it public now.